### PR TITLE
CHE-701: Extend debugger views from Window

### PR DESCRIPTION
**Pull request by vinokurig**
@vparfonov please review

### CODE REVIEW

File: plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/changevalue/ChangeValueViewImpl.java [hunk 1]
```diff
@@ -35,41 +37,47 @@
...
+                    @Override
+                    public void onClick(ClickEvent clickEvent) {
+                        delegate.onCancelClicked();
+                    }
+                });
+
+        changeButton = createButton(locale.changeValueViewChangeButtonTitle(), "debugger-change-value-change-btn", new ClickHandler() {
+            @Override
+            public void onClick(ClickEvent clickEvent) {
+                delegate.onCancelClicked();
```
Comment (vzhukovs) [inline plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/changevalue/ChangeValueViewImpl.java]: `.onCancelClicked();` ?

Comment (vinokurig) [inline plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/changevalue/ChangeValueViewImpl.java]: done, thanks for showing a mistake

File: plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/changevalue/ChangeValueViewImpl.ui.xml [hunk 2]
```diff
@@ -27,31 +22,14 @@
...
-                    </g:north>
-                    <g:center>
-                        <g:TextArea ui:field="value" width="415px" height="125px"/>
-                    </g:center>
-                </g:DockLayoutPanel>
-            </g:FlowPanel>
+            <g:TextArea ui:field="value" width="428px"/>
         </g:center>
     </g:DockLayoutPanel>
 </ui:UiBinder>
```
Comment (azatsarynnyy) [inline plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/changevalue/ChangeValueViewImpl.ui.xml]: add new line

Comment (vinokurig) [inline plugins/plugin-java/che-plugin-java-ext-debugger-java-client/src/main/java/org/eclipse/che/ide/ext/java/jdi/client/debug/changevalue/ChangeValueViewImpl.ui.xml]: done

**Comment by vparfonov** [comment]
ok

**Pull request merged**